embiggen/sequences/abstract_word2vec_sequence.py

```diff
@@ -49,7 +49,8 @@ def __init__(
             shuffle=shuffle,
             samples_number=self._sequences.samples_number,
             batch_size=batch_size,
-            elapsed_epochs=elapsed_epochs
+            elapsed_epochs=elapsed_epochs,
+            seed=seed
         )
 
     def on_epoch_end(self):
```

Comment:
Wthever to shuffle the vectors
spelling error in doc string, should be Whether

Reply:
Fixed!